docs/api-reference/tasks.md

```diff
@@ -1,6 +1,7 @@
 ---
 id: tasks
```

Comment:
Do we want to move this page over completely, or just the Tasks API section? There seems to be a lot in this doc that isn't API focused, like how the task lock system works. It actually seems that the Task API section points to the [API reference page](https://druid.apache.org/docs/latest/operations/api-reference.html#tasks) already, so this change may not be needed.